source/versioned-api/tests/crud-api-version-1-strict.yml

```diff
@@ -268,9 +268,7 @@ tests:
           - commandStartedEvent:
               command:
                 getMore: { $$type: [ int, long ] }
-                apiVersion: { $$exists: false }
-                apiStrict: { $$exists: false }
-                apiDeprecationErrors: { $$exists: false }
+                <<: *expectedApiVersion
```

Comment:
Should probably change the name of this test to reflect that `getMore` _does_ append API version